src/functionalTest/resources/scenarios/RIA-6692-internal-case-creation-ada-start-appeal.json

```diff
@@ -0,0 +1,59 @@
+{
+  "description": "RIA-6692 Internal case creation ADA & detained start appeal",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "AdminOfficer",
+    "input": {
+      "eventId": "startAppeal",
+      "state": "appealStartedByAdmin",
+      "caseData": {
+        "template": "minimal-appeal-started.json",
+        "replacements": {
+          "isAdmin": "Yes",
+          "appealReferenceNumber": "DRAFT",
+          "appellantInUk": "Yes",
+          "appellantInDetention": "Yes",
+          "isAcceleratedDetainedAppeal": "Yes",
+          "uploadTheAppealFormDocs": [
+            {
+              "id": "1",
+              "value": {
+                "document": {
+                  "document_url": "http://dm-store/documents/c78071ff-a011-42c1-99a2-17b0adb9b906",
+                  "document_filename": "some-appeal-form.pdf",
+                  "document_binary_url": "http://dm-store/documents/c78071ff-a011-42c1-99a2-17b0adb9b906/binary"
+                },
+                "description": "some appeal form description"
+              }
+            }
+          ]
+        }
+      }
+    }
+  },
+  "expectation": {
+    "status": 200,
+    "errors": [],
+    "caseData": {
+      "template": "minimal-appeal-started.json",
+      "replacements": {
+        "detentionStatus": "detainedAccelerated",
+        "tribunalDocuments": [
+          {
+            "id": "1",
+            "value": {
+              "document": {
+                "document_url": "$/http.+\/documents/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/",
+                "document_binary_url": "$/http.+\/documents/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}\/binary/",
+                "document_filename": "DRAFT-Talha Awan-appeal form1.pdf"
```

Comment:
This will be renamed again at Submit? Please put that in the submit ticket in comments, saying this needs to be renamed PostSubmit in SubmitAppeal.

Reply:
Added a comment into https://tools.hmcts.net/jira/browse/RIA-6694